core/arch/arm/kernel/boot.c

```diff
@@ -332,7 +333,9 @@ static void init_asan(void)
 #ifdef CFG_WITH_PAGER
 	asan_tag_access(__pageable_start, __pageable_end);
 #endif /*CFG_WITH_PAGER*/
+#ifndef CFG_DYN_CONFIG
 	asan_tag_access(__nozi_start, __nozi_end);
```

Comment:
MMU tables may still be allocated from the nozi section. Shouldn't it still be tagged for ASAN access in these cases?

Reply:
`__nozi_start` and `__nozi_end` doesn't exist with `CFG_DYN_CONFIG`. This is instead covered by the `boot_mem_init_asan()` call above.